### PR TITLE
Add comments to TokenValidationParameters describing parameters #201

### PR DESCRIPTION
**Pull request by brentschmaltz**
Add comments to TokenValidationParameters describing parameters #201
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/tushargupta51%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217%23issuecomment-130092053%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-08-11T22%3A07%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20c975a209122450fd322f41213eaadb50f8a1248a%20src/System.IdentityModel.Tokens/TokenValidationParameters.cs%20241%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217%23discussion_r36805026%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22nit%3A%20%27side%27-%3E%27site%27%22%2C%20%22created_at%22%3A%20%222015-08-11T22%3A02%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/System.IdentityModel.Tokens/TokenValidationParameters.cs%3AL481-490%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217%23issuecomment-130092053%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217%23discussion_r36805026%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/tushargupta51'></a>

- [ ] <a href='#crh-comment-Pull c975a209122450fd322f41213eaadb50f8a1248a src/System.IdentityModel.Tokens/TokenValidationParameters.cs 241'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217#discussion_r36805026'>File: src/System.IdentityModel.Tokens/TokenValidationParameters.cs:L481-490</a></b>
- <a href='https://github.com/tushargupta51'></a> nit: 'side'->'site'

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217?approve=0'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/217'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/System.IdentityModel.Tokens/TokenValidationParameters.cs [hunk 1]
```diff
@@ -457,7 +481,9 @@ public bool ValidateActor
 
         /// <summary>
         /// Gets or sets a boolean to control if the audience will be validated during token validation.
-        /// </summary>        
+        /// </summary>
+        /// <remarks>Validation of the audience, mitigates forwarding attacks. For example, a site that receives a token, could not replay it to another side.
```
Comment (tushargupta51) [inline src/System.IdentityModel.Tokens/TokenValidationParameters.cs]: nit: 'side'->'site'

**Comment by tushargupta51** [comment]
:shipit:

**Comment by brentschmaltz** [comment]
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/827af47924248cee0a7c44c51e8fa7722cde2015

**Pull request closed without merging**